src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+import java.io.Serializable;
```

Comment:
Hmm I didn't think of using Serializable, may I know it does/how it is used?

Reply:
Since we're allowed to define whatever format we want `duke.txt` to be in, I've decided to use Serializable, as I don't want to handle the tediousness of keeping the structure of the data (at least at the current iteration). By serializing my `Task` and `TaskList`, I can safely save to and read the binary format file that the JVM understands naturally (basically, I've outsourced maintaining the structure of the data file to Java itself).

_The official documentation is here: https://www.tutorialspoint.com/java/java_serialization.htm_

_While a good, small example can be found here: https://stackoverflow.com/questions/41746683/java-how-to-save-an-arraylist-into-a-file-and-open-it_

Hope that helps!